Gemfile

```diff
@@ -4,9 +4,15 @@ ruby "2.2.2"
 #
 # Bundle edge Rails instead: gem "rails", github: "rails/rails"
 gem "rails", "~>4.2"
-# Use sqlite3 as the database for Active Record
-gem "sqlite3", group: :development
-gem "pg", group: :production
+
+# Note: We're using sqllite3 for development and testing
+gem "sqlite3", group: [:development, :test]
```

Comment:
I think maybe no reasons for this, but sometimes the order of loading gems matter.